### ISSUE TITLE
Supressing a error is not working

### ISSUE THREAD

**Issue by Tapchicoma** [shyiko/ktlint#158]
I am trying to suppress `max-line-length` error, but it is not working.

What I've tried so far:
- adding `// ktlint-disable max-line-length` in the end of failing line
- adding `// ktlint-disable` in the end of failing line
- adding `/* ktlint-disable */` before failing line and `/* ktlint-enable */` after failing line

Ktlint version `1.15.0`

**Comment by Tapchicoma** [comment]
Ktlint version `1.15.1` has the same problem.

**Comment by Tapchicoma** [comment]
Happens only when project has `.editorconfig` with redefined `max-line-length`

**Comment by shyiko** [comment]
(Hopefully) Fixed in [0.16.0](https://github.com/shyiko/ktlint/releases/tag/0.16.0).

**Comment by shyiko** [comment]
@Zayankovsky any chance you can provide an example? I'm unable to reproduce.

**Comment by Zayankovsky** [comment]
This is my `.editorconfig`:
```
[*.{kt,kts}]
insert_final_newline=true
max_line_length=120
```
This class triggers `max-line-length` error, despite the suppression:
```
class Whatever {

    private fun whatever(whatever: Any) = whatever

    private val whatever = whatever(whatever(whatever(whatever(whatever(whatever(whatever(whatever(whatever("whatever"))))))))) // ktlint-disable max-line-length
}
```
This class does not trigger `max-line-length` error:
```
class Whatever {

    private fun whatever(whatever: Any) = whatever

    /* ktlint-disable max-line-length */
    private val whatever = whatever(whatever(whatever(whatever(whatever(whatever(whatever(whatever(whatever("whatever")))))))))
    /* ktlint-enable max-line-length */
}
```

**Comment by shyiko** [comment]
Thank you, @Zayankovsky! Fixed.

**Comment by westy92** [comment]
You can also use `@Suppress("ktlint:standard:max-line-length")`

https://pinterest.github.io/ktlint/0.49.1/faq/#how-do-i-suppress-errors-for-a-lineblockfile

**Comment by kenyee** [comment]
For some reason, I had to use @Suppress("ktlint:max-line-length") so it wouldn't get flagged via command line ktlint.
IDE ktlint is fine w/o the "ktlint:" prefix.  Both are roughly on the same version 0.46/0.48.

**Comment by paul-dingemans** [comment]
Please read docs and suppress "ktlint:standard:max-line-length"
